daemon/api.go

```diff
@@ -1526,6 +1526,11 @@ func getSnapConf(c *Command, r *http.Request, user *auth.UserState) Response {
 		var value interface{}
 		if err := tr.Get(snapName, key, &value); err != nil {
 			if config.IsNoOption(err) {
```

Comment:
I think it would make sense to push this forward into _tr.Get_ itself, but this PR sounds fine as a first step.

Reply:
I'll keep this in mind, will address it in a followup PR.